test/e2e/seeder/anvil-clients.ts

```diff
@@ -17,26 +17,26 @@ function createAnvilClients(instance: Instance, id: number, port: number) {
     id,
```

Comment:
from the docs: instances are accessible at `"http://localhost:8545/n"` so we need to pass an id


https://github.com/wevm/prool?tab=readme-ov-file#usage